book/src/concepts/resource.md

```diff
@@ -61,10 +71,20 @@ If you want to change a resource that is already inside of `Resources`:
 ```rust,no_run,noplaypen
 # extern crate amethyst;
 # use amethyst::ecs::{Resources};
-# struct MyResource;
+# struct MyResource {
+#   pub game_score: i32,
+# }
 # fn main() {
 #   let mut resources = Resources::new();
-let mut fetched = resources.try_fetch_mut::<MyResource>().expect("No MyResource present in Resources");
+#   let my = MyResource{
+#     game_score: 0,
+#   };
+#   resources.insert(my);
+  // try_fetch_mut returns a Option<FetchMut<MyResource>>
+  let mut fetched = resources.try_fetch_mut::<MyResource>()
+                             .expect("No MyResource present in Resources");
+  assert_eq!(fetched.game_score, 0);
+  fetched.game_score = 10;
```

Comment:
Could you add another assert_eq after this line showing that the score is now 10?
Thanks!

Reply:
Sure! I was debating between adding it or not :)